Makefile

```diff
@@ -1,7 +1,20 @@
+.EXPORT_ALL_VARIABLES:
+
 GOLANGCI_LINT_CONFIG = .golangci.yml
 IMAGE_REPOSITORY ?= app-sre
 QUAY_USER ?=
 QUAY_TOKEN ?=
+OPM_VERSION = v1.14.0
+
+BUNDLE_DEPLOY_DIR=deploy/bundle
+MANIFEST_DIR=$(BUNDLE_DEPLOY_DIR)/manifests
+CSV=$(MANIFEST_DIR)/deploymentvalidationoperator.clusterserviceversion.yaml
+OPERATOR_IMAGE=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
+OPEARTOR_IMAGE_TAG=v$(OPERATOR_VERSION)
```

Comment:
OPEARTOR? looks like typo...

Reply:
Suggestion. To avoid redefining what it has already been defined in boilerplate I would move `OPERATOR_IMAGE` and `OPERATOR_IMAGE_TAG` after the include and do it from the `OPERATOR_IMAGE_URI` as follows:

```bash
OPERATOR_IMAGE = $(shell echo ${OPERATOR_IMAGE_URI} | cut -d ":" -f 1)
OPERATOR_IMAGE_TAG = $(shell echo ${OPERATOR_IMAGE_URI} | cut -d ":" -f 2)

Reply:
In any case I think you will have to move them anyway since you need things like `IMAGE_REGISTRY` from boilerplate...

Reply:
My intention is to push those definitions into base boilerplate.  All the info is there, it's just in a form that we can't user without extraction logic like you offered.  There's no good reason to need to do that.  Just have more fundamentals in boilerplate and build up the existing vars from those fundamental vars.

Reply:
Fixed the typo

Reply:
I like the idea of moving those to `boilerplate/openshift/golang-osd-operator/standard.mk` and refactoring how `OPERATOR_IMAGE_URI`, so do it :)

Reply:
Pushed the definitions down into boilerplate